### PR TITLE
modify CSV if deployment is owned by CSV

### PR DESCRIPTION
**Pull request by maskarb**
Deployments owned by the CSV were not mounting onto the PVC. This PR fixes that.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,11 +80,8 @@ help:
 all: manager
 
 # Run tests
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
-	mkdir -p ${ENVTEST_ASSETS_DIR}
```
Comment (boaz0) [inline Makefile:85]: Why are we removing these? I thought it is essential to test Go operators?

**Review by boaz0 — APPROVED** [review]
All in all LGTM

**Review by abaiken — APPROVED** [review]
I tested the new bundle with an upgrade from 0.9.1 as well as deploying for development. Everything worked. Good work Michael!

File: Makefile (hunk 1 above)
Comment (maskarb) [reply Makefile:85]: These are not necessarily required. These assets were actually causing test failures, so I removed them. So long as Kubebuilder is properly installed, the tests will still run. If it turns out that we do need these, then they can be put back. But in the meantime, I think they can be removed.

**Pull request merged**